sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -142,7 +142,7 @@ public virtual EventData CreateEventFromMessage(AmqpMessage source)
 
         /// <summary>
         ///   Converts a given <see cref="AmqpMessage" /> response to a query for Event Hub properties into the
-        ///   corresponding <see cref="EventHubProperties" /> representation.
+        ///   corresponding <see cref="EventHubProperties" /> representation;
```

Comment:
Semicolon? Was this intentional?

Reply:
er... nope.   I must have had a bad search/replace at some point, because Caio had called out a few in weird spots in an earlier PR.